common/src/main/java/io/netty/util/concurrent/EventExecutorGroup.java

```diff
@@ -116,4 +116,16 @@
 
     @Override
     ScheduledFuture<?> scheduleWithFixedDelay(Runnable command, long initialDelay, long delay, TimeUnit unit);
+    
+    /**
+     * Set the {@link RejectTaskHandler} that will be called if the execution of a task fails. 
+     * 
+     * @param rejectTaskHandler the reject task handler that will be used for rejected tasks
+     */
+    void setRejectedTaskHandler(RejectedTaskHandler rejectTaskHandler);
+    
+    /**
+     * Returns the {@link RejectTaskHandler} that will be used for rejected tasks
```

Comment:
* Rename the parameter to `rejectedTaskHandler`.
* the reject task handler that will be used for rejected tasks -> the handler that handles rejected tasks